airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/client/AirbyteClient.kt

```diff
@@ -0,0 +1,97 @@
+package io.airbyte.cdk.load.client
+
+import io.airbyte.cdk.load.command.DestinationStream
+import io.airbyte.cdk.load.command.log
+import io.airbyte.cdk.load.data.AirbyteType
+import io.airbyte.cdk.load.orchestration.db.ColumnNameMapping
+import io.airbyte.cdk.load.orchestration.db.Sql
+import io.airbyte.cdk.load.orchestration.db.TableName
+
+abstract class AirbyteClient<DestinationDataType: Enum<DestinationDataType>>() {
```

Comment:
Can we use type parameters instead of words that look like classes? e.g. `T`

Reply:
To be discuss, here is my reasoning: I don't like using T that much. It is not explicit, and will be confusing if we need to add another generic. Having the IDE applying another color to the generic type helps understanding that this is not a type.